docs/html/development/index.rst

```diff
@@ -8,7 +8,8 @@ testing, and documentation.
 
 You can also join ``#pypa`` (general packaging discussion and user support) and
 ``#pypa-dev`` (discussion about development of packaging tools) `on Libera.chat`_,
-or the `distutils-sig mailing list`_, to ask questions or get involved.
+or the ``#pip`` channel on the `PyPA Discord`_, or the `distutils-sig mailing list`_,
+to ask questions or get involved.
```

Comment:
This is 100% a drive-by comment, but is the `distutils-sig` mailing list still active? AFAIU it is not.

Reply:
It's not active since most people moved to Discourse but it still has official channel status afaik.

Reply:
We made it read-only at some point.

Reply:
to clarity: we want to keep the mailing list link in?

Reply:
I think we can remove it here since you can’t “ask questions or get involved” on it anymore.

Reply:
> I think we can remove it here since you can’t “ask questions or get involved” on it anymore.

removed: [02330a2](https://github.com/pypa/pip/pull/12840/commits/02330a2f24379c70d8c59dfdb1589dd3c4b0efc7)